### ISSUE TITLE
update documentation for backdrop option

### ISSUE THREAD

**Issue by My6UoT9** [makeusabrew/bootbox#744]
Backdrop docu http://bootboxjs.com/documentation.html#bb-option-backdrop writes:
> Undefined (null) | The backdrop is displayed, but clicking on it has no effect.

In source code, the described behavior happens when backdrop is set to "static".  
see default being set to "static":
https://github.com/makeusabrew/bootbox/blob/master/bootbox.js#L134

And here it checks for "static":
https://github.com/makeusabrew/bootbox/blob/master/bootbox.js#L432

I suggest to change documentation too:

```
backdrop
Type: Boolean|String

Whether the dialog should be have a backdrop or not. Also determines whether clicking on the backdrop dismisses the modal.

Options:
"static"	The backdrop is displayed, but clicking on it has no effect.
true *	The backdrop is displayed, and clicking on it dismisses the dialog.
false	The backdrop is not displayed.
Default: "static"

* When this value is set to true, the dialog will only dismiss when onEscape is also set to true or some callback function.
```

**Comment by tiesont** [comment]
Probably worth doing. The docs are located in the gh_pages branch, so that sounds like a good pull-request. Otherwise, it will have to wait until I have a reason to update the docs (which usually only happens when a feature is added).

**Comment by tiesont** [comment]
Closed by #745
